### PR TITLE
Added thumbor_spaces plugin

### PR DESCRIPTION
**Pull request by siddhartham**

### CODE REVIEW

File: docs/plugins.rst [hunk 1]
```diff
@@ -170,6 +170,34 @@ Configuration in thumbor.conf:
...
+result storage.
+
+-  *URL:* https://github.com/siddhartham/thumbor_spaces
+-  *Installing:* ``pip install thumbor_spaces``
+
+Using it is simple, just change your configuration in thumbor.conf:
+
+::
+
+    # Use rackspace for result storage.
```
Comment (kkopachev) [inline docs/plugins.rst]: could you change `rackspace` here to `DigitalOcean Spaces`?

**Comment by heynemann** [comment]
Hey @siddhartham,

Thank you so much for your contribution, but we won't be merging it in thumbor's codebase. The reason for that is that we want to have a leaner codebase and this can easily be a different repository.

Actually it is encouraged that the community implements and maintains filters, loaders, storages and anything else that thumbor allows as extensions as separate projects. I'd also encourage you to do a Pull Request to the [thumbor's plugins](https://thumbor.readthedocs.io/en/latest/plugins.html) docs page to include your own plugin/filter/storage/loader, etc.

That way the community can benefit from it, without growing thumbor's codebase.

Once again, thank you very much for being a part of our community.

**Comment by siddhartham** [comment]
ohh my bad, i thought i submitted there

File: docs/plugins.rst (hunk 1 above)
Comment (siddhartham) [reply docs/plugins.rst]: done

**Comment by heynemann** [comment]
I'm really really sorry... I completely misunderstood your PR. I thought it was the actual filter. Really really sorry.

**Comment by heynemann** [comment]
Thank you very much for your contribution.

**Comment by kkopachev** [comment]
Hm, why does it need to be a separate plugin? DO Spaces support S3 API, so tc_aws should be able to handle it, right?

**Pull request merged**